lib/virtualGraphs.js

```diff
@@ -37,31 +37,45 @@ const update = (conn, name, mappings, options, params = {}) => {
   }).then(httpBody);
 };
 
-const remove = (conn, name, params = {}) => {
+const remove = (conn, name) => {
   const headers = conn.headers();
   return fetch(conn.request('admin', 'virtual_graphs', name), {
     method: 'DELETE',
     headers,
   }).then(httpBody);
 };
 
-const available = (conn, name, params = {}) => {
+const available = (conn, name) => {
   const headers = conn.headers();
   headers.set('Accept', 'application/json');
   return fetch(conn.request('admin', 'virtual_graphs', name, 'available'), {
     headers,
   }).then(httpBody);
 };
 
-const options = (conn, name, params = {}) => {
+const options = (conn, name) => {
   const headers = conn.headers();
   return fetch(conn.request('admin', 'virtual_graphs', name, 'options'), {
     headers,
   }).then(httpBody);
 };
 
-const mappings = (conn, name, params = {}) => {
+const mappings = (conn, name, requestOptions = {}) => {
   const headers = conn.headers();
+
+  if (requestOptions.preferUntransformed) {
```

Comment:
The TS definition for the requestOptions object contains the key `untransformed` instead of `preferUntransformed`. Which do we want?

Reply:
whoops, good catch. `preferUntransformed` was the one intended, so I'll update.